atomic_reactor/plugins/exit_sendmail.py

```diff
@@ -223,6 +169,71 @@ def _send_mail(self, receivers_list, subject, body):
             if s is not None:
                 s.quit()
 
+    def _get_email_from_koji_obj(self, obj):
+        if obj.get('krb_principal') and obj['krb_principal'] != '':
```

Comment:
Can be simplified to: `if obj.get('krb_principal'):`

Empty strings are falsy.